jsk_unitree_robot/jsk_unitree_startup/launch/get_location.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+
+  <arg name="network_interface" default="wlan0" />
+  <arg name="google_map_location_key_path" default="/var/lib/robot/google_map_location_key.yaml" />
+
+  <node name="location_node"
```

Comment:
how about move this codne on jsk_robot_common ?

Reply:
Thanks! I moved it to jsk_robot_startup